client/reader/tags/trending-tags.tsx

```diff
@@ -20,17 +22,18 @@ const trackTagClick = ( slug: string ) => {
 	} );
 };
 
-const TagRow = ( props: TagRowProps ) => (
-	<div className="trending-tags__column" key={ props.slug }>
-		<a
-			href={ `/tag/${ encodeURIComponent( props.slug ) }` }
-			onClick={ trackTagClick.bind( null, props.slug ) }
-		>
-			<span className="trending-tags__title">{ titlecase( props.title ) }</span>
-			<span className="trending-tags__count">{ formatNumberCompact( props.count ) }</span>
-		</a>
-	</div>
-);
+const TagRow = ( props: TagRowProps ) => {
+	const locale = getLocaleSlug();
+	const path = addLocaleToPathLocaleInFront( `/tag/${ encodeURIComponent( props.slug ) }`, locale );
```

Comment:
One thing im noticing here is this is adding the locale for both logged in and logged out versions of the page.

So if I:
* am logged in
* goto my user settings and change locale to spanish
* goto logged-in tags page and click a link at the top

it navigates me to `/es/tag/{tagSlug}` as well. Im not sure how much of an issue it is as things still seem to work, but maybe we want to limit adding the locale if we are logged in?

Reply:
Good catch! hmmm I just realised there's something else missing in this PR, when logged in, we are supposed to redirect the language out of routes e.g. always use /tag/{tagSlug} and use the users chosen language from the user settings. I'll look into fixing these two things 👀

https://github.com/Automattic/wp-calypso/pull/78656#pullrequestreview-1505210317